texts/formatted_aroundTheWorld.txt

```diff
@@ -0,0 +1,1747 @@
+;	"time" n sg
+;		"in" pr
+"<.>"
+	"." sent
+"<While>"
+;	"while" adv
+;	"while" cnjsub
+	"while" cnjadv
+;	"while" n sg
+"<disembarking>"
+;	"disembark" vblex pprs
+;	"disembark" vblex ger
+	"disembark" vblex subs
+"<in>"
+;	"in" adv
+	"in" pr
+"<Egypt>"
+	"Egypt" np top sg
+"<,>"
+	"," cm
+"<they>"
+	"prpers" prn subj p3 mf pl
+"<are>"
+	"be" vbser pres
+;	"be" vbser pres p1 pl
+;	"be" vbser pres p2 sg
+;	"be" vbser pres p2 pl
+;	"be" vbser pres p3 pl
+"<watched>"
+;	"watch" vblex pp
+	"watch" vblex past
+"<by>"
+	"by" adv
+;	"by" pr
+"<a>"
+	"a" det ind sg
+"<Scotland>"
+	"Scotland" np top sg
+"<Yard>"
+	"yard" n sg
+"<detective>"
+	"detective" n sg
+"<,>"
+	"," cm
+"<Detective>"
+	"detective" n sg
+"<Fix>"
+	"fix" n sg
+;	"fix" vblex inf
+;	"fix" vblex pres
+;	"fix" vblex imp
+"<,>"
+	"," cm
+"<who>"
+;	"who" prn itg mf sp
+	"who" prn rel an mf sp
+"<has>"
+;	"have" vbhaver pres p3 sg
+	"have" vblex pres p3 sg
+"<been>"
+	"be" vbser pp
+"<dispatched>"
+;	"dispatch" vblex pp
+	"dispatch" vblex past
+"<from>"
+	"from" pr
+"<London>"
+;	"London" np top sg
+	"London" np cog sg
+"<in>"
+	"in" adv
+;	"in" pr
+"<search>"
+;	"search" n sg
+;	"search" vblex inf
+	"search" vblex pres
+;	"search" vblex imp
+"<of>"
+	"of" pr
+"<a>"
+	"a" det ind sg
+"<bank>"
+	"bank" n sg
+;	"bank" vblex inf
+;	"bank" vblex pres
+;	"bank" vblex imp
+"<robber>"
+	"robber" n sg
+"<.>"
+	"." sent
+"<Since>"
+;	"since" adv
+	"since" cnjsub
+;	"since" pr
+"<Fogg>"
+	"*Fogg"
+"<fits>"
+;	"fit" n pl
+	"fit" vblex pres p3 sg
+"<the>"
+	"the" det def sp
+"<vague>"
+	"vague" adj
+"<description>"
+	"description" n sg
+"<Scotland>"
+	"Scotland" np top sg
+"<Yard>"
+	"yard" n sg
+"<was>"
+;	"be" vbser past p1 sg
+	"be" vbser past p3 sg
+"<given>"
+;	"given" adj
+	"give" vblex pp
+;	"given" n sg
+"<of>"
+	"of" pr
+"<the>"
+	"the" det def sp
+"<robber>"
+	"robber" n sg
+"<,>"
+	"," cm
+"<Detective>"
+	"detective" n sg
+"<Fix>"
+	"fix" n sg
+;	"fix" vblex inf
+;	"fix" vblex pres
+;	"fix" vblex imp
+"<mistakes>"
+;	"mistake" n pl
+	"mistake" vblex pres p3 sg
+"<Fogg>"
+	"*Fogg"
+"<for>"
+;	"for" cnjadv
+	"for" pr
+"<the>"
+	"the" det def sp
+"<criminal>"
+;	"criminal" adj
+	"criminal" n sg
+"<.>"
+	"." sent
+"<Since>"
+;	"since" adv
+	"since" cnjsub
+;	"since" pr
+"<he>"
+	"prpers" prn subj p3 m sg
+"<cannot>"
+	"not" adv
+		"can" vbmod pres
+"<secure>"
+;	"secure" adj
+;	"secure" vblex inf
+	"secure" vblex pres
+;	"secure" vblex imp
+"<a>"
+	"a" det ind sg
+"<warrant>"
+	"warrant" n sg
+"<in time>"
+	"in time" adv
+;	"time" n sg
+;		"in" pr
+"<,>"
+	"," cm
+"<Fix>"
+	"fix" n sg
+;	"fix" vblex inf
+;	"fix" vblex pres
+;	"fix" vblex imp
+"<boards>"
+;	"board" n pl
+	"board" vblex pres p3 sg
+"<the>"
+	"the" det def sp
+"<steamer>"
+	"steamer" n sg
+"<(>"
+	"(" lpar
+"<the>"
+	"the" det def sp
+"<Mongolia>"
+	"Mongolia" np top sg
+"<)>"
+	")" rpar
+"<conveying>"
+	"convey" vblex pprs
+;	"convey" vblex ger
+;	"convey" vblex subs
+"<the>"
+	"the" det def sp
+"<travelers>"
+	"*travelers"
```

Comment:
This should be added into the dictionary as an alternative spelling to "travellers".